develop/integrations/wallets.md

```diff
@@ -1,4 +1,60 @@
 ---
 title: Wallets
-description: TODO
----
\ No newline at end of file
+description: Explore blockchain wallets: securely manage digital assets with hot wallets for online access or cold wallets for offline, enhanced security.
+---
+
+# Wallets
+
+## What is a Blockchain Wallet?
+
+A wallet serves as your gateway to interacting with blockchain networks. Rather than storing funds, wallets secure your private keys, controlling access to your blockchain assets. Your private key provides complete control over all permitted transactions on your blockchain account, making it essential to keep it secure.
+
+Wallet types fall into two categories based on their connection to the internet:
+
+- [**Hot Wallets**](#hot-wallets) - online storage through websites, browser extensions or smartphone apps
+- [**Cold Wallets**](#cold-wallets) - offline storage using hardware devices or air-gapped systems
+
+## Hot Wallets
+
+<div class="grid cards" markdown>
+
+-   :material-clock-fast:{ .lg .middle } __Talisman__
+
+    ---
+
+    A better way to explore Web3. Keep your assets safe, manage your portfolio, and explore Polkadot and Ethereum apps with Talisman. Interact with Web3 apps, store your favorite crypto assets, and manage your accounts on over 150+ Substrate and EVM networks. NFT Management, ledger Support, fiat On-ramp, portfolio tracking.
+
+    [:octicons-arrow-right-24: Reference](https://talisman.xyz/){target=\_blank}
+
+-   :fontawesome-brands-markdown:{ .lg .middle } __Subwallet__
+
+    ---
+
+    A non-custodial Polkadot, Substrate & Ethereum wallet. Track, send, receive, and monitor multi-chain assets on 150+ networks. Import account with seed phrase, private key, QR code, and JSON file. Import token & NFT, attach read-only account. XCM Transfer, NFT Management, Parity Signer & Ledger support, light clients support, EVM DApp support, MetaMask compatibility, custom endpoints, fiat on-ramp, phishing detection, transaction history.
```

Comment:
Up to you, but I'd remove the `substrate` word here to avoid any confusion with the word Polkadot:
```suggestion
    A non-custodial Polkadot & Ethereum wallet. Track, send, receive, and monitor multi-chain assets on 150+ networks. Import account with seed phrase, private key, QR code, and JSON file. Import token & NFT, attach read-only account. XCM Transfer, NFT Management, Parity Signer & Ledger support, light clients support, EVM DApp support, MetaMask compatibility, custom endpoints, fiat on-ramp, phishing detection, transaction history.
```